images/image-user/cloudbuild-images.yaml

```diff
@@ -0,0 +1,57 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
```

Comment:
I think this file has to be called `cloudbuild.yaml` and we need one for each image (in a dedicated directory)

Reply:
I can change the name and add it to images sub-directory. Do we need a file per image? Can it be called with the flag `--substitutions=_IMAGE_TYPE="username-group"`, changing the type for each image?

Reply:
It might work, yeah. The image builder (like `gcr.io/k8s-testimages/image-builder:v20200901-ab141a0`) runs a `/builder` binary, which supports the following CLI flags:
```
bash-4.4# ./builder -h
Usage of ./builder:
  -allow-dirty
        If true, allow pushing dirty builds.
  -build-dir string
        If provided, this directory will be uploaded as the source for the Google Cloud Build run.
  -env-passthrough string
        Comma-separated list of specified environment variables to be passed to GCB as substitutions with an _ prefix. If the variable doesn't exist, the substitution will exist but be empty.
  -gcb-config string
        If provided, this will be used as the name of the Google Cloud Build config file. (default "cloudbuild.yaml")
  -log-dir string
        If provided, build logs will be sent to files in this directory instead of to stdout/stderr.
  -no-source
        If true, no source will be uploaded with this build.
  -project string
        If specified, use a non-default GCP project.
  -scratch-bucket string
        The complete GCS path for Cloud Build to store scratch files (sources, logs).
  -variant string
        If specified, build only the given variant. An error if no variants are defined.
  -with-git-dir
        If true, upload the .git directory to GCB, so we can e.g. get the git log and tag.
```

Reply:
Could use `-env-passthrough` to pass `IMAGE_TYPE` flag which means that can use the one cloud build file creating the different images for `image-user`.  The `-gcb-config` flag means can pass the cloud build filename. There should be able to keep names `cloudbuild-images.yaml` and `cloudbuild-manifests.yaml`. What do you think @saschagrunert?

Reply:
Sounds good to me :+1: